source/Projectile.cpp

```diff
@@ -48,7 +48,15 @@ Projectile::Projectile(const Ship &parent, Point position, Angle angle, const We
 	cachedTarget = TargetPtr().get();
 	if(cachedTarget)
 		targetGovernment = cachedTarget->GetGovernment();
-	double inaccuracy = weapon->Inaccuracy();
+	
+	double inaccuracy = 0;
+	if (weapon->SpinupTime() > 1)
+	{
+		inaccuracy = spinupProgress * weapon->SpinupInaccuracy() + (1 - spinupProgress) * weapon->Inaccuracy();
+	}
+	else
+		inaccuracy = weapon->Inaccuracy();
```

Comment:
Looks clean.  I was trying to segregate it to avoid extra work but this is fine as well.  Also, I think you meant to say weapons that get more accuracte with spinup have negative spinup inaccuracy, not the ones that get more inaccurate?  If that's how this is going to be done then there needs to be a sanity check for `spinupInaccuracy > inaccuracy`.

Reply:
There should be no need for a sanity check.

Reply:
I said that badly.  `if(-spinupInaccuracy > inaccuracy)` (if spinupInaccuracy is larger in the negative direction than inaccuracy) you're going to have inaccuracy on the negative side, which may or may not cause problems for the `Angle::Random()` but will definitely be conceptually odd.  Maybe you can have a sweet spot in the middle of the spinup?  But why?

Reply:
Why not?
Angle::Random takes the absolute value of the input, so it'll be fine.

Reply:
Ok yeah but I just thought about that as I was typing it.  Do we want this sweet spot possibility to exist?  In fairness, I've been a proponent of stat flexibility but I just don't see how it would make sense for a weapon to ever do this.

Reply:
Doesn't mean we should disallow it, particularly when it would be more work to disallow it than to just allow it to be a possibility.

Reply:
My own logic, coming back to bite me...heh